src/conf_mode/interfaces-dummy.py

```diff
@@ -55,7 +55,10 @@ def generate(dummy):
     return None
 
 def apply(dummy):
-    d = DummyIf(dummy['ifname'])
+    if 'netns' in dummy:
+        d = DummyIf(ifname=dummy['ifname'], netns=dummy['netns'])
```

Comment:
You could shorten this and drop the if statement. Given that I expect this will need to be implemented for all interfaces eventually, this might be a more readable/shorter option
```suggestion
    d = DummyIf(ifname=dummy['ifname'], netns=dummy.get('netns', None))
```